pkg/readers/adapters/amqp.go

```diff
@@ -241,37 +251,35 @@ func NewAMQPAdapter() (ReadAdapter, error) {
 	}, nil
 }
 
-func NewAmqpConfig() (amqpAdapter.Config, error) {
-	// @TODO this needs to be extracted to its own method
+func generateConsumerID() (string, error) {
```

Comment:
I know usually you are not propagating the errors. But in this case the execution could be interrupted.